src/test/java/org/prebid/server/bidder/somoaudience/SomoaudienceBidderTest.java

```diff
@@ -161,11 +165,11 @@ public void makeHttpRequestShouldReturnTwoHttpRequestsWhenTwoImpsHasDifferentPla
         final BidRequest bidRequest = BidRequest.builder()
                 .imp(asList(Imp.builder()
                                 .banner(Banner.builder().build())
-                                .ext(Json.mapper.valueToTree(ExtPrebid.of(null, ExtImpSomoaudience.of("placement1"))))
+                                .ext(mapper.valueToTree(ExtPrebid.of(null, ExtImpSomoaudience.of("placement1", null))))
                                 .build(),
                         Imp.builder()
-                                .banner(Banner.builder().build())
-                                .ext(Json.mapper.valueToTree(ExtPrebid.of(null, ExtImpSomoaudience.of("placement2"))))
+                                .video(Video.builder().build())
```

Comment:
I suspect this test is not correct, basically it is obsolete as bidder implementation doesn't group imps with same placement hash in one request anymore.
Next one is also obsolete